app/src/main/java/com/kickstarter/viewmodels/projectpage/ProjectPageViewModel.kt

```diff
@@ -1150,6 +1178,25 @@ interface ProjectPageViewModel {
                 .subscribe {
                     backingViewGroupIsVisible.onNext(false)
                 }.addToDisposable(disposables)
+
+            continuePledgeFlow
```

Comment:
@jlplks add tests on `ProjectPageViewModelTest` that covers this new logic! you will likely need for this piece two different tests to cover the if/else branches as adding a new user to current user requires a mock on the environment current user object 🤔